app/moves/middleware/set-results.moves.js

```diff
@@ -19,11 +19,13 @@ function setResultsMoves(bodyKey, locationKey) {
           moves: activeMoves,
           context: locationKey === 'from_location' ? 'incoming' : 'outgoing',
           showLocations: !req.location,
+          currentLocationType: req.session.currentLocation.location_type,
```

Comment:
Same here for using `req.location` instead of the current session. And maybe might be worth updating `currentLocationType` to `locationType`.